cmd/kube-apiserver/app/server.go

```diff
@@ -275,7 +245,7 @@ func Run(s *options.APIServer) error {
 	genericConfig.SupportsBasicAuth = len(s.BasicAuthFile) > 0
 	genericConfig.Authorizer = authorizer
 	genericConfig.AdmissionControl = admissionController
-	genericConfig.APIResourceConfigSource = apiResourceConfigSource
+	genericConfig.APIResourceConfigSource = storageFactory.APIResourceConfigSource
```

Comment:
Fair point. When genericConfig.APIResourceConfigSource and storageFactory.APIResourceConfigSource diverge for us, we will have to set it to different value.